Watch Extension/HomeView.swift

```diff
@@ -25,6 +25,9 @@ struct HomeView: View {
                 LoggedOutHomeView()
             }
         }
+        .onAppear() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/bf2b870c3cf32d3c360c3a79cc833555b46225e4/Watch Extension/HomeView.swift#L28"></span>When using trailing closures, empty parentheses should be avoided after the method call.<br />
<code>empty_parentheses_with_trailing_closure</code> <code>HomeView.swift:28</code></td>
    </tr>
  </tbody>
</table>